src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +70,11 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(p -> p.getAge() >= fromAge && ((p.getAge() <= femaleToAge
+                        && p.getSex() == Person.Sex.WOMAN)
```

Comment:
better firstly check sex and then age:
```suggestion
                .filter(p -> p.getAge() >= fromAge && ((p.getSex() == Person.Sex.WOMAN
                        && p.getAge() <= femaleToAge)
```